### ISSUE TITLE
AbstractHttpTransport disconnects unnecessarily, seems to ignore "maxSessionsPerBrowser"

### ISSUE THREAD

**Issue by pfantrax** [cometd/cometd#612]
We recently tried upgrading from cometd 2.9.1 to cometd 3.0.6.

After upgrading, when loading two tabs with our cometd application in the same browser, we noticed that as soon as the second tab connects, the previous tab is disconnected.

This code in AbstractHttpTransport seems to be causing the problem. (We are using long polling)

```
if (session != null && !session.getId().equals(message.getClientId()))
{
    session.disconnect();
    return;
}
```

We were always able to maintain simultaneous connections with cometd 2.9.1

We have also confirmed the same behaviour when using the bundled cometd jquery sample.

This happens no matter what we choose for the "maxSessionsPerBrowser" parameter?

Any ideas? Is this the intended behaviour?

**Comment by sbordet** [comment]
Not intended behavior, I think it's a regression introduced by #608.
Will investigate, stay tuned.

**Comment by sbordet** [comment]
This is now fixed in branch "3.0.x". Can you please try it out to confirm it's working for you ?
